### PR TITLE
New UI port and 2FA

### PR DESCRIPTION
**Pull request by gsanchietti**
Card: https://trello.com/c/AotUwjir/317-new-ui-port

Merge after https://github.com/NethServer/nethsecurity/pull/318

### CODE REVIEW

File: remote_access.rst [hunk 1]
```diff
@@ -23,35 +26,99 @@ Such credentials can be used to login into the web interface or using SSH.
...
+
+Restricting access to NethSecurity UI
+-------------------------------------
+
+By default, this interface is accessible on port 9090 from both your internal network (LAN) and the wider internet (WAN).
+While convenient, this can potentially introduce a security risk.
+
+To mitigate this risk, you can take two main steps:
+
+- remove the ``Allow-UI-from-WAN`` rule: go to the Firewall rules page,  navigate to the ``Input rules`` tab,
```
Comment (filippocarletti) [inline remote_access.rst:43]:
```suggestion
- remove the ``Allow-UI-from-WAN`` rule: go to the Firewall rules page, navigate to the ``Input rules`` tab,
```

File: remote_access.rst [hunk 2]
```diff
@@ -23,35 +26,99 @@ Such credentials can be used to login into the web interface or using SSH.
...
+
+Both URL are accessible from LAN and WAN by default.
+
+Restricting access to NethSecurity UI
+-------------------------------------
+
+By default, this interface is accessible on port 9090 from both your internal network (LAN) and the wider internet (WAN).
+While convenient, this can potentially introduce a security risk.
+
+To mitigate this risk, you can take two main steps:
```
Comment (filippocarletti) [inline remote_access.rst:41]:
```suggestion
To mitigate this risk, you have two options (remove or restrict access):
```

File: remote_access.rst [hunk 3]
```diff
@@ -23,35 +26,99 @@ Such credentials can be used to login into the web interface or using SSH.
...
+
+- remove the ``Allow-UI-from-WAN`` rule: go to the Firewall rules page,  navigate to the ``Input rules`` tab,
+  and locate the "Allow-UI-from-WAN" rule. Click the :guilabel:`Delete` button to remove it
+- allowing access from specific IPs or networks: in the Firewall rules page, locate the "Allow-UI-from-WAN"
```
Comment (filippocarletti) [inline remote_access.rst:45]:
```suggestion
- restrict access from specific IPs or networks: in the Firewall rules page, locate the "Allow-UI-from-WAN"
```

File: remote_access.rst [hunk 4]
```diff
@@ -23,35 +26,99 @@ Such credentials can be used to login into the web interface or using SSH.
...
+  rule and click the :guilabel:`Edit` button. In the ``Source address``  field, enter the IP addresses or network CIDRs
+  from which you want to allow access to the NethSecurity UI.
 
-* NethSecurity UI (``ns-ui``): A custom user interface focused on usage simplicity.
-* LuCI: The original OpenWrt web interface. Please bear in mind that some pages may cause unpredictable configuration changes.
+  For example, to allow access only from your home network, you could enter the IP range 192.168.1.0/24
```
Comment (filippocarletti) [inline remote_access.rst:49]:
```suggestion
  For example, to allow access only from your home network, you could enter the 192.168.1.0/24 network.
```

File: remote_access.rst [hunk 5]
```diff
@@ -23,35 +26,99 @@ Such credentials can be used to login into the web interface or using SSH.
...
 Change web user interface port
 ------------------------------
 
-Web user interfaces commonly operate on port 443, a convention established for its ease of accessibility through modern web browsers.
-However, specific scenarios necessitate the redirection of port 443 to an internal web server.
+Users have the capability to change the NethSecurity UI port.
 
-Users have the capability to selectively disable or enable both default interfaces: the ns-ui and LuCI. Moreover, NethSecurity allows the
-addition of extra ``ns-ui`` instances on different ports, granting users the freedom to customize their network configurations according to their needs.
+To change NethSecurity UI port from 9090 to 8181, execute: ::
```
Comment (filippocarletti) [inline remote_access.rst:63]:
```suggestion
To change the NethSecurity UI port from 9090 to 8181, execute: ::
```

File: remote_access.rst [hunk 6]
```diff
@@ -23,35 +26,99 @@ Such credentials can be used to login into the web interface or using SSH.
...
 
-To disable both UIs on port 443, and enable enable ``ns-ui`` only port 9090, execute: ::
+  uci set ns-ui.config.nsui_extra_port=8181
+  uci commit ns-ui && ns-ui
+
+Disable web user interface on port 443
+--------------------------------------
+
+While exposing port 443 (HTTPS) can be necessary for certain services, directly accessing the NethSecurity UI through this port
+may introduces a potential security risk. Here's how to safely maintain port 443 functionality while protecting your NethSecurity UI.
```
Comment (filippocarletti) [inline remote_access.rst:72]:
```suggestion
may introduce a potential security risk. Here's how to safely maintain port 443 functionality while protecting your NethSecurity UI.
```

File: remote_access.rst [hunk 7]
```diff
@@ -23,35 +26,99 @@ Such credentials can be used to login into the web interface or using SSH.
...
+
+If you don't require accessing the NethSecurity UI through port 443, disable it to minimize attack opportunities.
+Execute the following command on your NethServer system: ::
```
Comment (filippocarletti) [inline remote_access.rst:75]:
```suggestion
Execute the following commands on your NethServer system: ::
```

**Review by filippocarletti — CHANGES_REQUESTED** [review]

**Review by filippocarletti — APPROVED** [review]

**Review by cotosso — APPROVED** [review]
Everything seems fine to me!

**Pull request merged**